ui/articles-aem-renderer/build.gradle

```diff
@@ -5,14 +5,24 @@ android {
 }
 
 dependencies {
+    implementation project(':library:xml-model')
 
-    implementation "android.arch.lifecycle:extensions:${deps.androidLifecycle}"
-    implementation "android.arch.persistence.room:runtime:${deps.roomVersion}"
-    implementation "com.android.support:appcompat-v7:${deps.androidSupport}"
-
+    // For Testing
     androidTestImplementation "com.android.support.test:runner:${deps.androidTestingSupport}"
     androidTestImplementation "android.arch.persistence.room:testing:${deps.roomVersion}"
+    testImplementation "android.arch.core:core-testing:${deps.androidLifecycle}"
```

Comment:
this dependency does nothing since it is targeting JVM unit tests and not the android tests.